docs/user/lib_use_cases_lib_mapper.md

```diff
@@ -39,6 +39,14 @@ net_con = NTC(host=device.name, username="demo", password="secret", device_type=
 Another use case could be using an example like the above in an Ansible filter. That would allow you to write a filter utilizing whichever automation library you needed without having to store the driver for each one in your Source of Truth.
 
 
+## Aerleon Mapper
+
+--8<-- "docs/user/lib_mapper/aerleon.md
```

Comment:
since these are new, wouldn't this PR have the actual changed feels for each of these source md files?

Reply:
The files exist, them being included here does not exists